app/src/main/java/ru/vssemikoz/newsfeed/MainApplication.java

```diff
@@ -32,18 +32,18 @@ public void onCreate() {
         initNewsDataBase(this);
     }
 
-    void initRetrofit(){
+    void initRetrofit() {
         retrofit = new Retrofit.Builder()
                 .baseUrl(MAIN_URL)
                 .addConverterFactory(GsonConverterFactory.create())
                 .build();
     }
 
-    void initNewsApi(){
+    void initNewsApi() {
         newsApi = retrofit.create(NewsApi.class);
     }
 
-    void initNewsDataBase(Context context){
+    void initNewsDataBase(Context context) {
```

Comment:
The database is usually a single word, not composite: data base. So, it's better to call related stuff `Database`.